Gemfile

```diff
@@ -43,6 +43,7 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 # Reduces boot times through caching; required in config/boot.rb
```

Comment:
### Optional
- [ ] In our seeds.rb file, we are not able to load the data because our Author field is missing `email` and `password` fields. Since this is not part of the requirements, I would like to give you a heads up